modules/admin_manual/pages/configuration/user/oidc/ms-azure-setup.adoc

```diff
@@ -143,8 +143,8 @@ If you want to let ownCloud create users which are not present during a OIDC aut
     ],
 ----
 
-////
-=== Register ownCloud Clients
+=== Register ownCloud Desktop and Mobile Clients
 
 To allow the ownCloud clients (Web/desktop/Android/iOS) to interact with the identity provider, you have to register them as clients.
-////
+
+Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. As prerequisite, an ownCloud Enterprise subscription including full branding is needed. For more details get in touch with {oc-support-url}[ownCloud Support].
```

Comment:
```suggestion
Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. As prerequisite, an ownCloud full branding subscription is needed. For more details get in touch with {oc-support-url}[ownCloud Support].
```
@michaelstingl corrected, pls check